src/templates/React InstantSearch widget/package.json.hbs

```diff
@@ -54,6 +54,7 @@
     "@types/node": "15.3.0",
     "@types/react": "17.0.0",
     "@types/react-dom": "17.0.0",
+    "@types/react-instantsearch-core": "6.26.0",
     "@types/react-instantsearch-dom": "6.10.0",
```

Comment:
should this one not updated too? or is it not automatically released when its dependency updated?

Reply:
Since there weren't any change for `react-instantsearch-dom` it didn't justify changing the version number, which requires us to explicitly add `@types/react-instantsearch-core` to get its latest version in the template.